#goal
uint32 cmdType
string setMode
geometry_msgs/Pose goalPose
---
#result
bool success
---
#feedback
geometry_msgs/Pose dronePose
